Index: templates/Hassan.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!-- Extending the predefined project layout -->\r\n{% extends \"layouts/base.html\" %}\r\n\r\n<!- This variable sets name of browser tab  -->\r\n{% set project = \"Hassan\" %}\r\n\r\n<!- This is a replacement block for Body in base.html -->\r\n{% block body %}\r\n    <!- Try building a new body here ->\r\n    <p>hey im hassan whats popping</p>\r\n{% endblock %}\r\n\r\n<!- This is turning off birds background from base.html -->\r\n{% block background %}\r\n{% endblock %}\r\n
===================================================================
diff --git a/templates/Hassan.html b/templates/Hassan.html
--- a/templates/Hassan.html	
+++ b/templates/Hassan.html	
@@ -7,7 +7,7 @@
 <!- This is a replacement block for Body in base.html -->
 {% block body %}
     <!- Try building a new body here ->
-    <p>hey im hassan whats popping</p>
+    <p>what's popping i'm hassan </p>
 {% endblock %}
 
 <!- This is turning off birds background from base.html -->
Index: templates/Abby.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!-- Extending the predefined project layout -->\r\n{% extends \"layouts/base.html\" %}\r\n\r\n<!- This variable sets name of browser tab  -->\r\n{% set project = \"Abby\" %}\r\n\r\n<!- This is a replacement block for Body in base.html -->\r\n{% block body %}\r\n    <!- Try building a new body here ->\r\n    <p>Howdy im abby!</p>\r\n{% endblock %}\r\n\r\n<!- This is turning off birds background from base.html -->\r\n{% block background %}\r\n{% endblock %}\r\n
===================================================================
diff --git a/templates/Abby.html b/templates/Abby.html
--- a/templates/Abby.html	
+++ b/templates/Abby.html	
@@ -7,7 +7,7 @@
 <!- This is a replacement block for Body in base.html -->
 {% block body %}
     <!- Try building a new body here ->
-    <p>Howdy im abby!</p>
+    <p>Howdy i'm abby!</p>
 {% endblock %}
 
 <!- This is turning off birds background from base.html -->
Index: templates/Evan.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!-- Extending the predefined project layout -->\r\n{% extends \"layouts/base.html\" %}\r\n\r\n<!- This variable sets name of browser tab  -->\r\n{% set project = \"Evan\" %}\r\n\r\n<!- This is a replacement block for Body in base.html -->\r\n{% block body %}\r\n    <!- Try building a new body here ->\r\n    <p>salutations im  evan!</p>\r\n{% endblock %}\r\n\r\n<!- This is turning off birds background from base.html -->\r\n{% block background %}\r\n{% endblock %}\r\n
===================================================================
diff --git a/templates/Evan.html b/templates/Evan.html
--- a/templates/Evan.html	
+++ b/templates/Evan.html	
@@ -7,7 +7,7 @@
 <!- This is a replacement block for Body in base.html -->
 {% block body %}
     <!- Try building a new body here ->
-    <p>salutations im  evan!</p>
+    <p>salutations i'm  evan!</p>
 {% endblock %}
 
 <!- This is turning off birds background from base.html -->
