Index: templates/Hassan.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\n<!-- Extending the predefined project layout -->\n{% extends \"layouts/base.html\" %}\n\n<!- This variable sets name of browser tab  -->\n{% set project = \"Hassan\" %}\n\n<!- This is a replacement block for Body in base.html -->\n{% block body %}\n<!- Try building a new body here ->\n<div class=\"center-block\" style=\"margin: 2% 5% 5%; text-align:left; font-family: gilroy,serif;\">\n\n    <div class=\"col-12\">\n        <p>Hello, {{name}}!</p>\n    </div>\n\n    <div class=\"col-12\">\n        <form action=\"/greet\" id=\"greet\" method=\"POST\" >\n            <label for=\"name\">Type a greeting name?</label>\n            <input name=\"name\" id=\"name\">\n            <input type=\"submit\" value=\"Greetings!\">\n        </form>\n    </div>\n\n\n\n</div>\n{% endblock %}\n\n<!- This is turning off birds background from base.html -->\n{% block background %}\n{% endblock %}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/Hassan.html b/templates/Hassan.html
--- a/templates/Hassan.html	
+++ b/templates/Hassan.html	
@@ -15,7 +15,7 @@
     </div>
 
     <div class="col-12">
-        <form action="/greet" id="greet" method="POST" >
+        <form action="/Hassan" id="greet" method="POST" >
             <label for="name">Type a greeting name?</label>
             <input name="name" id="name">
             <input type="submit" value="Greetings!">
Index: templates/layouts/greet.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/layouts/greet.html b/templates/layouts/greet.html
new file mode 100644
--- /dev/null	
+++ b/templates/layouts/greet.html	
@@ -0,0 +1,14 @@
+<div class="center-block" style="margin: 2% 5% 5%; text-align:left; font-family: gilroy,serif;">
+  <div class="col-12">
+    <p>Hello, {{name}}!</p>
+  </div>
+
+  <div class="col-12">
+    <form action="/greet" id="greet" method="POST" >
+      <label for="name">Type a greeting name?</label>
+      <input name="name" id="name">
+      <input type="submit" value="Greetings!">
+    </form>
+  </div>
+
+</div>
