Index: templates/nilf.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!-- Extending the predefined project layout -->\n{% extends \"layouts/base.html\" %}\n\n<!- This variable sets name of browser tab  -->\n{% set project = \"Greetings\" %}\n\n<!- This is a replacement block for Body in base.html -->\n{% block body %}\n    <!- Try building a new body here ->\n    whats popping!\n{% endblock %}\n\n<!- This is turning off birds background from base.html -->\n{% block background %}\n{% endblock %}\n
===================================================================
diff --git a/templates/nilf.html b/templates/nilf.html
--- a/templates/nilf.html	
+++ b/templates/nilf.html	
@@ -7,7 +7,8 @@
 <!- This is a replacement block for Body in base.html -->
 {% block body %}
     <!- Try building a new body here ->
-    whats popping!
+<img src="https://lostinanime.com/wp-content/uploads/2018/02/Miira-no-Kaikata-06-29.jpg" alt="Miira no Kaikata - 06 - 29 - Lost in Anime" jsname="HiaYvf" jsaction="load:XAeZkd;" class="n3VNCb" data-noaft="1" style="width: 400px; height: 225px; margin: 0px;">
+
 {% endblock %}
 
 <!- This is turning off birds background from base.html -->
